migrations/advanced_permissions_phase_2.go

```diff
@@ -71,19 +71,19 @@ func (worker *Worker) runAdvancedPermissionsPhase2Migration(lastDone string) (bo
 
 	if progress.CurrentTable == "TeamMembers" {
 		// Run a TeamMembers migration batch.
-		if result := <-worker.app.Srv.Store.Team().MigrateTeamMembers(progress.LastTeamId, progress.LastUserId); result.Err != nil {
-			return false, progress.ToJson(), result.Err
+		if result := worker.app.Srv.Store.Team().MigrateTeamMembers(progress.LastTeamId, progress.LastUserId); result != nil {
+			return false, progress.ToJson(), result
 		} else {
-			if result.Data == nil {
+			if result == nil {
 				// We haven't progressed. That means that we've reached the end of this stage of the migration, and should now advance to the next stage.
 				progress.LastUserId = strings.Repeat("0", 26)
 				progress.CurrentTable = "ChannelMembers"
 				return false, progress.ToJson(), nil
 			}
 
-			data := result.Data.(map[string]string)
-			progress.LastTeamId = data["TeamId"]
-			progress.LastUserId = data["UserId"]
+			// data := result.Data.(map[string]string)
```

Comment:
probably needs to be uncommented and returned as @jespino pointed out.

Reply:
Thank you @ali-farooq0 , let me make the changes and push again